cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -93,7 +93,7 @@ func LogOrWriteConfig(fileName string, cfg *config.KubeSchedulerConfiguration, c
 	}
 
 	if klogV.Enabled() {
-		klogV.Info("Using component config", "\n-------------------------Configuration File Contents Start Here---------------------- \n", buf.String(), "\n------------------------------------Configuration File Contents End Here---------------------------------\n")
+		klogV.InfoS("Using component config", "config", buf.String())
```

Comment:
I think there was some discussion about line for multi line logging. Have it been resolved, or we are just fixing this due to logcheck? 

If we haven't resolved the multi line discussion it would be better to not mark this directory for logcheck.

Reply:
sg

Reply:
doesn't this escape the contents of the config file, making it ~unreadable?

Reply:
No, I added more readable formatting of multi-line strings in https://github.com/kubernetes/klog/pull/273

The commit message in this PR shows what the updated log output looks like in this case here: https://github.com/kubernetes/kubernetes/pull/108159/commits/0f4d0660a7674f816d4f60791b1cfe191fab7050

Reply:
ah, cool